### DISCUSSION TITLE
https://fastapi.tiangolo.com/learn/ is showing 404 for last 1 hr since today..

### DISCUSSION THREAD

**Discussion by anil-patro** [fastapi/fastapi#11379] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
NA
```

### Description

1. Go to Fast website and check any links like - learn ... 
2. It says 404 in most of the links.
3. Seeing other lanuages help when I am trying to search english

### Operating System

macOS

### FastAPI Version

latest website

### Pydantic Version

NA

### Python Version

NA

### Additional Context

NA

**Comment by aaoobd3** [comment]
please fix it

**Comment by aaoobd3** [comment]
yesterday too

**Comment by JavierSanchezCastro** [comment, marked as answer]
It's actually working for me

**Comment by fabianfalon** [comment]
Is working
https://fastapi.tiangolo.com/learn/

**Comment by anil-patro** [comment]
Working fine. Closing the discussion, thanks for all the help
